ledger/sandbox/BUILD.bazel

```diff
@@ -401,17 +401,34 @@ server_conformance_test(
     ],
 )
 
+server_conformance_test(
+    name = "conformance-test-contract-id-seeding",
```

Comment:
Is this an extra set of tests only for weak seeding?

Reply:
By default Sandbox runs without seeding (on the contrary of kvutils), this test Sandbox works with seeding.